xtask/src/main.rs

```diff
@@ -63,6 +63,26 @@ impl FromStr for Memory {
     }
 }
 
+#[derive(Clone)]
+enum DramSize {
+    TwoG,
+    FourG,
+    EightG,
+}
+
+impl FromStr for DramSize {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "2g" | "2G" => Ok(Self::TwoG),
```

Comment:
```suggestion
            "2g" | "2G" => Ok(Self::TwoG),
```

If you decide to keep the `to_lowercase` conversion above, you don't need to match the uppercase versions (since they'll never be present).